velox/docs/configs.rst

```diff
@@ -697,6 +697,10 @@ Spark-specific Configuration
      - integer
      -
      - The current task's Spark partition ID. It's set by the query engine (Spark) prior to task execution.
+   * - spark.legacy_time_parser_policy
+     - bool
+     - false
+     - If true, simple date formatter is used for time formatting and parsing. Strict date formatter is used by default.
```

Comment:
Would it be possible to provide some examples to illustrate the differences and / or add links to reading materials that explain this further?

Reply:
Thanks for your suggestion. I've added some examples and an issue link that contains more of the differences.